tendrl-cephd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Tendrl Ceph Daemon to Manage ceph tasks
```

Comment:
Description should be more meaningful. May be like "Tendrl ceph daemon to manage ceph storage system"